azext_iot/operations/events3/_events.py

```diff
@@ -131,75 +193,87 @@ def _output_msg_kpi(msg):
-            interface_context = pnp_context['interface'].get(msg_interface_id)
+            msg_schema = str(
+                msg.application_properties.get(b"iothub-message-schema"), "utf8"
+            )
+            interface_context = pnp_context["interface"].get(msg_interface_id)
             if interface_context:
                 msg_schema_context = interface_context.get(msg_schema)
                 if msg_schema_context:
-                    msg_context_display = msg_schema_context.get('display')
-                    msg_context_unit = msg_schema_context.get('unit')
+                    msg_context_display = msg_schema_context.get("display")
+                    msg_context_unit = msg_schema_context.get("unit")
 
                     if msg_context_display:
-                        event_source['event']['payload'] = {}
-                        event_source['event']['payload'][msg_context_display] = payload
+                        event_source["event"]["payload"] = {}
+                        event_source["event"]["payload"][msg_context_display] = payload
                         if msg_context_unit:
-                            event_source['event']['payload']['unit'] = msg_context_unit
-
-        if 'anno' in properties or 'all' in properties:
-            event_source['event']['annotations'] = unicode_binary_map(msg.annotations)
-        if 'sys' in properties or 'all' in properties:
-            if not event_source['event'].get('properties'):
-                event_source['event']['properties'] = {}
-            event_source['event']['properties']['system'] = system_props
-        if 'app' in properties or 'all' in properties:
-            if not event_source['event'].get('properties'):
-                event_source['event']['properties'] = {}
-            app_prop = msg.application_properties if msg.application_properties else None
+                            event_source["event"]["payload"]["unit"] = msg_context_unit
+
+        if "anno" in properties or "all" in properties:
+            event_source["event"]["annotations"] = unicode_binary_map(msg.annotations)
+        if "sys" in properties or "all" in properties:
+            if not event_source["event"].get("properties"):
+                event_source["event"]["properties"] = {}
+            event_source["event"]["properties"]["system"] = system_props
+        if "app" in properties or "all" in properties:
+            if not event_source["event"].get("properties"):
+                event_source["event"]["properties"] = {}
+            app_prop = (
+                msg.application_properties if msg.application_properties else None
+            )
 
             if app_prop:
-                event_source['event']['properties']['application'] = unicode_binary_map(app_prop)
-        if output.lower() == 'json':
+                event_source["event"]["properties"]["application"] = unicode_binary_map(
+                    app_prop
```

Comment:
is this new break following some new linter rule?

Reply:
This is autoformatting with python black.